metaflow/plugins/kubernetes/kubernetes_job.py

```diff
@@ -58,7 +61,183 @@ def __init__(self, client, **kwargs):
         self._client = client
         self._kwargs = kwargs
 
-    def create(self):
+    def create_job_spec(self):
+        client = self._client.get()
```

Comment:
has to create a sub-function called `create_job_spec` so we could reuse the jobspec created for K8s jobs and plug that in directly into Jobsets.